helm/superset/templates/_helpers.tpl

```diff
@@ -85,13 +101,8 @@ SQLALCHEMY_TRACK_MODIFICATIONS = True
 
 class CeleryConfig:
   imports  = ("superset.sql_lab", )
-  {{- if .Values.supersetNode.connections.redis_password }}
-  broker_url = f"redis://:{env('REDIS_PASSWORD')}@{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
-  result_backend = f"redis://:{env('REDIS_PASSWORD')}@{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
-  {{- else }}
-  broker_url = f"redis://{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
-  result_backend = f"redis://{env('REDIS_HOST')}:{env('REDIS_PORT')}/0"
-  {{- end }}
+  broker_url = f"{CELERY_REDIS_URL}"
+  result_backend = f"{CELERY_REDIS_URL}"
```

Comment:
no need for f-strings here

Reply:
🤦🏻‍♂️ - Missed while code refactoring.. Will fix this

Reply:
Done... Please check now